src/GildedRose.elm

```diff
@@ -16,7 +16,7 @@ updateQuality =
 
 updateQualityItem (Item name sellIn quality) =
     let
-        quality' =
+        quality_ =
```

Comment:
as the reason for deprecating `'`s was to force better names i guess we should aim for something like `updatedQuality`. What do you think?